back/src/answer/answer.model.ts

```diff
@@ -0,0 +1,23 @@
+import { UserDTO } from 'src/user/model/user.model';
+import { Field, Int, ObjectType } from '@nestjs/graphql';
+
+@ObjectType('Answer')
+export class AnswerDTO {
+  @Field()
+  id: string;
+
+  @Field(() => UserDTO, { nullable: true })
```

Comment:
Type is not required
```suggestion
  @Field({ nullable: true })
```